### PR TITLE
Updated the help message for the "Package" field of the Bake stage

### PR DESCRIPTION
**Pull request by vjames-bmc**
Since you can bake multiple debian packages, the help text has been updated to reflect that.

packagebakehelp

### CODE REVIEW

File: app/scripts/modules/core/help/helpContents.js [hunk 1]
```diff
@@ -152,7 +152,8 @@ module.exports = angular.module('spinnaker.core.help.contents', [])
     'pipeline.config.quickPatchAsg.skipUpToDate': '<p>Skip instances which already have the requested version.</p>',
     'pipeline.config.jenkins.propertyFile': '<p>(Optional) Configures the name to the Jenkins artifact file used to pass in properties to later stages in the Spinnaker pipeline.</p>',
     'pipeline.config.bake.package': '<p>The name of the package you want installed (without any version identifiers).</p>' +
-      '<p>If your build produces a deb file named "myapp_1.27-h343", you would want to enter "myapp" here.</p>',
+      '<p>If your build produces a deb file named "myapp_1.27-h343", you would want to enter "myapp" here.</p>' +
+      '<p>If there are multiple packages (space separated), then they wil be installed in the order they are entered.</p>',
```
Comment (duftler) [inline app/scripts/modules/core/help/helpContents.js]: s/wil/will

**Comment by duftler** [comment]
The issue here is that multiple packages are only supported in the OSS bakery (rosco). So the help message needs to show the more detailed message only if the `roscoMode` flag is enabled. Maybe define two help key/message pairs and choose the one to display based on true/false of `roscoMode`? Would need to do this in `aws/bakeStage.html` and `gce/bakeStage.html`.

**Comment by anotherchrisberry** [comment]
Since this is specific to OSS bakery, it probably makes more sense to move the existing help text to the `netflix` module help registry (https://github.com/spinnaker/deck/blob/master/app/scripts/modules/netflix/help/netflixHelpContents.registry.js) - then we don't have to worry about toggling it elsewhere.

**Comment by duftler** [comment]
Even better. I had forgotten about netflixHelpContents.

**Comment by vjames-bmc** [comment]
Thanks @duftler and @anotherchrisberry . I will make the suggested changes.

**Comment by vjames-bmc** [comment]
Will close this PR and create another one with the suggested changes

**Pull request closed without merging**